test/new/db/json/json4

```diff
@@ -3,8 +3,8 @@ pdbj
 pdfj
 pDj
 pdj
-# pDJ BROKEN
-# pdJ BROKEN
+pDJ 31
```

Comment:
without an argument this thing should work fine

Reply:
ok 74081eb9577e1d4aa0e0d942cc7f35e6fd17cef1